app/workers/downstream_live_worker.rb

```diff
@@ -1,11 +1,22 @@
+require 'sidekiq-unique-jobs'
+
 class DownstreamLiveWorker
   attr_reader :content_item_id, :web_content_item, :payload_version, :message_queue_update_type, :update_dependencies
 
   include DownstreamQueue
   include Sidekiq::Worker
   include PerformAsyncInQueue
 
-  sidekiq_options queue: HIGH_QUEUE
+  sidekiq_options queue: HIGH_QUEUE,
+                  unique: :until_executing,
+                  unique_args: :uniq_args
+
+  def self.uniq_args(args)
+    [
+      args.first.fetch("content_item_id"),
+      'live'
+    ]
+  end
```

Comment:
Sorry I'm a bit late on this.

I assume this is the unique criteria for the worker so if there's another item with this content_item_id it doesn't get queued - ignore me if I'm wrong.

We should also have message_queue_update_type and update_dependencies in there. For example if we had an item queued as the result of a link update it wouldn't update dependencies and just send a update_type of links, however if it had a major update while sat in the queue this message wouldn't get sent out or dependencies updated.